config/DECam/processCcdWithFakes.py

```diff
@@ -0,0 +1,36 @@
+# This file is modeled after HSC's, and has all of the minimal DECam-specific
+# calibrate configs. Since processCcdWithFakesTask uses a minimal version of
+# calibrateTask, we can't import the whole thing, because it configs for
+# astrometry and photometry; these are turned off in processCcdWithFakes.
+
+import os.path
+
+from lsst.meas.algorithms import ColorLimit
+from lsst.meas.astrom import MatchOptimisticBConfig
+
+# HACK: Throw away any changes imposed by obs configs, especially plugins.
+config.loadFromString(type(config)().saveToString())
+
+obsConfigDir = os.path.join(os.path.dirname(__file__))
+# calFile = os.path.join(ObsConfigDir, "calibrate.py")
```

Comment:
Any reason not to delete these commented out lines if we don't need them?

Reply:
Good point, can delete